tests/acceptance/features/apiSearchContent/contentSearch.feature

```diff
@@ -265,3 +265,39 @@ Feature: content search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario: check extracted properties of a file from project space
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file "filesForUpload/testimage.mp3" to "testimage.mp3" in space "new-space"
+    And user "Alice" has uploaded a file "filesForUpload/testavatar.jpg" to "testavatar.jpg" in space "new-space"
+    When user "Alice" gets the following extracted properties of resource "testimage.mp3" inside space "new-space" using the WebDAV API
```

Comment:
```suggestion
    When user "Alice" gets the following extracted properties of resource "testaudio.mp3" inside space "new-space" using the WebDAV API
```
Isn't that audio file?

Reply:
Yes, it's an audio file.

Reply:
Please rename the file name to `testaudio.mp3` so that will not create confusions.